### PR TITLE
[T4A5][T13-B3] Sean Zhang

### PR DESCRIPTION
**Pull request by puzzledsean**
Referenced code from #693 and #1250 for some guidance

Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/AddressBook.java [hunk 1]
```diff
@@ -22,7 +25,8 @@
 
     private final UniquePersonList allPersons;
     private final UniqueTagList allTags; // can contain tags not attached to any person
-
+    private final List<Tagging> taggingHistory = new ArrayList<Tagging>(); //contains the history of all the tagging
```
Comment (z11i) [inline src/seedu/addressbook/data/AddressBook.java:28]: Put the initialization in the constructor method.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        this.tag = tag;
+        isTagged = true;
+    }
+    
+    public void deleteTagging() {
+        isTagged = false;
+    }
+    
+    public String printTagging() {
+        String result = (isTagged) ? ADD_TAGGING : DELETE_TAGGING;
```
Comment (z11i) [inline src/seedu/addressbook/data/tag/Tagging.java:24]: Succinct ternary operator usage here.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
```
Comment (z11i) [inline src/seedu/addressbook/data/tag/Tagging.java:5]: Write comments about what this class does.

**Comment by z11i** [comment]
Comments added. Good job overall!

**Comment by puzzledsean** [comment]
Appreciate the comments/feedback!

**Pull request closed without merging**